src/language/walker/blockScopeAwareRuleWalker.ts

```diff
@@ -21,13 +21,15 @@ import { IOptions } from "../rule/rule";
 import { isBlockScopeBoundary } from "../utils";
 import { ScopeAwareRuleWalker } from "./scopeAwareRuleWalker";
 
+// tslint:disable:deprecation
```

Comment:
can you comment here explaining why we're disabling the rule for the file?

Reply:
done